Add fix command (#1627)
VirtusLab/scala-cli: website/docs/commands/fix.md

Reviewer: Some example with directives spread between multiple files would be useful.
An example output `project.scala` could then be included, plus there'd be no reason to ignore the `'''bash` snippets in `docs-tests`

---
title: Fix
sidebar_position: 20
---

> The fix commands can be used to perform fixes on the Scala CLI project.

Reviewer: ```suggestion
The fix command can be used to perform fixes on a Scala CLI project.
```

## Migrating directives

> It is recommended to keep all using directives centralized in one file. Fix command along with `--migrate-directives` option will migrate all directives of given inputs to the `project.scala` file (if the file does not exist, Scala CLI will create it).

Reviewer: ```suggestion
It is recommended to keep all `using` directives centralised in one file. Fix command along with the `--migrate-directives` option will migrate all directives of given inputs to the `project.scala` configuration file (if the file does not exist, Scala CLI will create it in the project root directory).
```

> Following command will move all directives inside current directory to `project.scala` file:

Reviewer: ```suggestion
The following command will move all directives inside the current working directory to the `project.scala` file:
```

> ```

Reviewer: ````suggestion
```bash ignore
````

scala-cli fix --migrate-directives .
```

> Please note that this is equivalent to:

Reviewer: ```suggestion
Note that this is equivalent to:
```

```bash ignore

> scala-cli migrate-directives .

Reviewer: is this example still relevant?
since we're not adding the `migrate-directives` alias
